testsuite/forge/src/backend/k8s/cluster_helper.rs

```diff
@@ -677,12 +690,25 @@ pub async fn create_management_configmap(kube_namespace: String, keep: bool) ->
             ..ObjectMeta::default()
         },
     };
-    configmap.create(&PostParams::default(), &config).await?;
-
-    info!(
-        "Created configmap {} with data {:?}",
-        management_configmap_name, data
-    );
+    if let Err(KubeError::Api(api_err)) = configmap.create(&PostParams::default(), &config).await {
+        if api_err.code == 409 {
+            info!(
+                "Configmap {} already exists, continuing with it",
```

Comment:
Thank you for fixing this. I was meaning to put out this change but you beat me to it :)

Reply:
oops :p well Forge needs lots of improvement -- more opportunities!